### PR TITLE
Keyboard selection of symbols/pairs, i18n caching

### PR DESCRIPTION
**Pull request by SaenkoM**
Symbols/pairs selects: refactoring, keyboard selection, indication of picked items
Removed i18n caching

### CODE REVIEW

File: src/components/ExportDialog/ExportDialog.js [hunk 1]
```diff
@@ -24,32 +24,35 @@ import ExportTargetsSelector from './ExportTargetsSelector'
...
 
-  onTargetSelect = (target) => {
-    if (!this.handlers[target]) {
-      this.handlers[target] = () => {
-        const { currentTargets } = this.state
-        if (!currentTargets.includes(target)) {
-          this.setState({ currentTargets: [...currentTargets, target] })
-        }
-      }
+  componentWillReceiveProps(nextProps) {
```
Comment (jarindr) [inline src/components/ExportDialog/ExportDialog.js]: componentWillReceiveProps is deprecated :/

Comment (jarindr) [reply src/components/ExportDialog/ExportDialog.js]: just an idea how to change to getDerivedStateFromProps ( didn't try to run 😆 )
```js
getDerivedStateFromProps(nextProps, prevState) {
    if(nextProps.type !== prevState.type) {
      return  { currentTargets: [nextProps.type] }
   }
   return { type: nextProps.type }
}
```

Comment (SaenkoM) [reply src/components/ExportDialog/ExportDialog.js]: thanks! always forgetting `componentWillReceiveProps` will be gone soon, such a useful method... :(

Comment (jarindr) [reply src/components/ExportDialog/ExportDialog.js]: yeah and getDerivedStateFromProps need a bit of hack to get prevProps LOL

**Pull request merged**